_posts/2021-12-13-kafka-segment-retention.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+title: "Deep dive into Apache Kafka storage internals: segments, rolling and retention"
+date: 2021-12-13
+author: paolo_patierno
+---
+
+Everyone who uses Apache Kafka knows that it behaves as a commit-log when it comes to deal with storing records.
```

Comment:
Maybe we can word it without the _everyone knows_ ... it is a bit passive-agressive towards people who might not know ;-)

```suggestion
Apache Kafka behaves as a commit-log when it comes to deal with storing records.
```

Reply:
Good advice, thanks!